lib/benchee/formatters/html.ex

```diff
@@ -91,8 +91,17 @@ defmodule Benchee.Formatters.HTML do
 
     suite
   end
-  def output(_suite) do
-    raise "You need to specify a file to write the HTML to in the configuration as formatter_options: [html: [file: \"my.html\"]]"
+
+  @default_filename "benchmark_output/my.html"
+  def output(suite) do
+    suite
+    |> update_filename()
+    |> output()
+  end
+
+  defp update_filename(suite) do
+    updated_configuration = %Configuration{suite.configuration | formatter_options: %{html: %{:file => @default_filename}}}
```

Comment:
Just a small style thing here, but we use the `%{file: @default_filename}` style when we're dealing with maps that have atoms as keys.

Reply:
makes sense.